jenkins/webhook-proxy/pipeline.template.json

```diff
@@ -2,24 +2,24 @@
     "kind": "BuildConfig",
     "apiVersion": "v1",
     "metadata": {
-        "name": "NAME",
+        "name": "{{.Name}}",
         "creationTimestamp": null
     },
     "spec": {
         "triggers": [
             {
                 "type": "Generic",
                 "generic": {
-                    "secret": "TRIGGER_SECRET"
+                    "secret": "{{.TriggerSecret}}"
                 }
             }
         ],
         "runPolicy": "Serial",
         "source": {
             "type": "Git",
             "git": {
-                "uri": "GIT_URI",
-                "ref": "BRANCH"
+                "uri": "{{.GitURI}}",
+                "ref": "{{.Branch}}"
             },
             "sourceSecret": {
                 "name": "cd-user-with-password"
```

Comment:
@metmajer @clemensutschig This is the template used to render the `BuildConfig` which represents the Jenkins pipeline. So any request to the webhook proxy which wants to create/start a pipeline needs to supply enough information to fill those slots. Further, we probably need to send some build args to Jenkins as well, right?